src/expressionEvaluator/jsExpr.ts

```diff
@@ -0,0 +1,5 @@
+import { ExpressionNodeType } from './expressionNodeType'
+
+export interface JSExpr {
```

Comment:
this is rather an ExpressionNode ?

Reply:
we can even add this to the same file expressionNode (that includes this and the ExpressionNodeType)

Reply:
uhm, right, I'll do it